cmds/record.c

```diff
@@ -132,8 +132,14 @@ char * get_libmcount_path(struct opts *opts)
 
 #ifdef INSTALL_LIB_PATH
 	snprintf(lib, PATH_MAX, "%s/%s", INSTALL_LIB_PATH, libmcount);
-	if (access(lib, F_OK) != 0 && errno == ENOENT)
-		pr_warn("Didn't you run 'make install' ?\n");
+	if (access(lib, F_OK) == 0)
+		return lib;
+
+	if (errno == ENOENT)
+		pr_warn("Library does not exist in %s. Didn't you run 'make install' ?\n"
+			, INSTALL_LIB_PATH);
```

Comment:
Just a style issue - please put the comma after the string literal.

Reply:
I also think that this message doesn't look good to me.